extensions/spring-xd-extension-http/Californium.properties

```diff
@@ -0,0 +1,41 @@
+#Californium CoAP Properties file
```

Comment:
I think this CoAP properties file intended to be checked in with a different PR ;)